packages/pulp-rpm/pulp-rpm.spec

```diff
@@ -204,8 +204,8 @@ Group: Development/Languages
 Requires: python-pulp-rpm-common = %{pulp_version}
 Requires: python-pulp-oid_validation >= 2.7.0
 Requires: pulp-server = %{pulp_version}
-Requires: createrepo >= 0.9.9-21
 Requires: createrepo_c >= 0.4.1-1
+Requires: python-createrepo_c
```

Comment:
@rohanpm, python-createrepo_c doesn't seem to be available in Fedora, python2-createrepo_c does (as well as for epel7)

Reply:
@goosemania yeah that's right, but python2-createrepo_c "Provides" python-createrepo_c, e.g.

    [root@aed3906cb029 /]# rpm -q python2-createrepo_c --provides
    python-createrepo_c = 0.10.0-15.fc27
    python-createrepo_c(x86-64) = 0.10.0-15.fc27
    python2-createrepo_c = 0.10.0-15.fc27
    python2-createrepo_c(x86-64) = 0.10.0-15.fc27

Is that fine or do I need to introduce some %if case here?